docs/walkthroughs/local-setup.adoc

```diff
@@ -17,17 +17,18 @@ for more detailed guides.
 >= 2.3.2.0
 * https://hub.docker.com/[Docker Hub account]
 * https://docs.docker.com/engine/installation/[Docker]
-* https://github.com/openshift/origin/releases[oc command line client]
+* https://github.com/openshift/origin/releases[oc command line client] >= 3.7.0-rc0
```

Comment:
I'd even remove this requirement as the installer will pull down the correct version of oc and put it in /usr/local/bin. This sound right to you @aidenkeating? Would you agree?

Reply:
@david-martin Yep that's correct

Reply:
Ok, I understand how it's supposed to work, so this can be removed.
(However, I'll add an issue where the installer will run an existing older `oc` instead of the downloaded one, if it appears earlier in the PATH)

Reply:
Just thinking, since this may replace and existing version of `oc` it should at least warn the user beforehand

Reply:
@mmurphy Great point! If you're creating that issue, you can put a link to it in this PR.
I'll remove that bullet from the doc and merge this PR